eclipse/src/saros/session/resources/validation/messages.properties

```diff
@@ -0,0 +1,3 @@
+ResourceChangeValidator_0=Participants with read-only access should not edit the resources of this shared project. If you wish to makes changes, ask the host to give you write-access.
+ResourceChangeValidator_1=You should leave the Saros session before deleting a shared project. Deleting a project has no effect on the other session participants, so the session will become inconsistent.
+ResourceChangeValidator_2=You should leave the Saros session before renaming or moving a shared project. Renaming or moving a project has no effect on the other session participants, so the session will become inconsistent.
```

Comment:
Have not tried out but I guess this is not true. Looking at the delta visitor moving the project also moves the files and files moves counts as delete if they are moved to a different  project. Same applies when deleting a project.

Reply:
Ok it seems to some logic issues that is not the case, however the log statement is more than confusing.

DEBUG 22:37:23,627 [ModalContext] (SharedResourcesManager.java:235) ignoring delta changes for project P/B as it was only opened

The project B was not opened, it was closed and does not exists anymore ...

Reply:
> Have not tried out but I guess this is not true. Looking at the delta visitor moving the project also moves the files and files moves counts as delete if they are moved to a different project. Same applies when deleting a project.

Do you mean case 2 will never apply? I am not really sure what you are trying to test.

Reply:
All cases apply. My issue was if I delete project A, what will happen on the remote side, e.g will only inconsistencies occur or will the whole content (not the project itself) be deleted as well.

Reply:
We should test the case, but if I remember correctly we already tested the case in our last release testing and had the expected behavior of an inconsistent state without deleting the resources on host side.